packages/formation/sass/site/site.scss

```diff
@@ -1,32 +1,33 @@
 // Variables
 
-$font-path: "~uswds/src/fonts";
-$image-path: "~uswds/src/img";
-@import "../core";
-@import "../modules/m-alert";
-@import "../modules/m-breadcrumbs";
-@import "m-crisis-line";
-@import "../modules/m-table";
-@import "../modules/m-button";
-@import "../modules/m-form-elements";
-@import "../modules/m-search";
-@import "../modules/m-loading-indicator";
-@import "m-vet-nav";
-@import "m-footer";
-@import "../modules/m-print";
-@import "../modules/m-dropdown";
-@import "../modules/m-nav-sidebar";
-@import "../modules/m-overlay";
-@import "../modules/m-additional-info";
-@import "../mobile-typography";
-@import "../shame";
-@import "../modules/m-modal";
-@import "../modules/m-progress-bar";
-@import "../modules/va-pagination";
-@import "../modules/va-tabs";
-@import "../modules/m-process-list";
-@import "color-palette";
-@import "../icons";
-@import "../modules/m-megamenu";
-@import "../modules/m-hub-page-link-list";
-@import "../modules/m-promo-banner";
+$font-path: '~uswds/src/fonts';
+$image-path: '~uswds/src/img';
+@import '../core';
+@import '../modules/m-alert';
+@import '../modules/m-breadcrumbs';
+@import 'm-crisis-line';
+@import '../modules/m-table';
+@import '../modules/m-button';
+@import '../modules/m-form-elements';
+@import '../modules/m-search';
+@import '../modules/m-loading-indicator';
+@import 'm-vet-nav';
+@import 'm-footer';
+@import '../modules/m-print';
+@import '../modules/m-dropdown';
+@import '../modules/m-nav-sidebar';
+@import '../modules/m-overlay';
+@import '../modules/m-additional-info';
+@import '../mobile-typography';
+@import '../shame';
+@import '../modules/m-modal';
+@import '../modules/m-progress-bar';
+@import '../modules/va-pagination';
+@import '../modules/va-tabs';
+@import '../modules/m-process-list';
+@import 'color-palette';
+@import '../icons';
+@import '../modules/m-megamenu';
+@import '../modules/m-hub-page-link-list';
+@import '../modules/m-promo-banner';
+@import '../modules/m-responsive-tables';
```

Comment:
I put this in here for the same reason. Not sure why we have two similar files like this.

Reply:
Huh. Good call. We'll have to investigate this later. :male_detective: